priority/management/commands/calculate-priority.py

```diff
@@ -0,0 +1,121 @@
+from django.core.management.base import BaseCommand, CommandError
+from sqlalchemy import create_engine, Table
+import numpy as np
+import pandas as pd
+from datetime import date, timedelta
+from project.models import Project
+from funding.models import Attribution
+from priority.models import slurm_priority
+import sys
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        #number of QOS levels
+        QOS_levels=4.0
+        Today=date.today()
+        #engine for reading/writing sql files
+        # sqlite://<nohostname>/<pathtofile>
+        engine = create_engine('sqlite:///db.sqlite3', echo=False)
```

Comment:
Does this still work if Django is configured to use a different database location, or different database manager?